lang/en.yml

```diff
@@ -14,8 +14,12 @@ en:
   SilverStripe\CMS\Controllers\CMSMain:
     ACCESS: "Access to '{title}' section"
     ACCESS_HELP: 'Allow viewing of the section containing page tree and content. View and edit permissions can be handled through page specific dropdowns, as well as the separate "Content permissions".'
+    ADDEDTODRAFTSHORT: 'Draft'
```

Comment:
Just revert changes to en.yml - this file will be automatically added when we run https://github.com/silverstripe/silverstripe-tx-translator

Reply:
Updated